tests/phpunit/tests/rest-api/test-notification-controller.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Unit tests covering WP_REST_Notification_Controller functionality.
+ */
+class WP_Test_REST_Notification_Controller extends WP_Test_REST_Post_Type_Controller_Testcase {
+	public function test_register_routes() {
+		$this->markTestSkipped( 'TODO Implement' );
+	}
+
+	public function test_context_param() {
+		$this->markTestSkipped( 'TODO Implement' );
+	}
+
+	public function test_get_items() {
+		$this->markTestSkipped( 'TODO Implement' );
+	}
+
+	public function test_get_item() {
+		$this->markTestSkipped( 'TODO Implement' );
+	}
+
+	public function test_create_item() {
+		$this->markTestSkipped( 'TODO Implement' );
+	}
+
+	public function test_update_item() {
+		$this->markTestSkipped( 'TODO Implement' );
+	}
+
+	public function test_delete_item() {
+		$this->markTestSkipped( 'TODO Implement' );
+	}
+
+	public function test_prepare_item() {
+		$this->markTestSkipped( 'TODO Implement' );
+	}
```

Comment:
@Sephsekla These are required by the rest test interface. We will fill out this test suite as we build out the rest-api.

Reply:
Good call. I'd like to make sure we add each test at the same time as its associated functionality.

Reply:
@Sephsekla I totally agree. That is why I added tests for the schema. We could easily get behind on testing if we don't stay on top of it.